common/tile.cpp

```diff
@@ -796,48 +826,40 @@ static bool tile_info_pollution(char *buf, int bufsz,
      "Hills (Coals)"
      "Hills (Coals) [Pollution]"
  */
-const char *tile_get_info_text(const struct tile *ptile,
+const char *tile_get_info_text(const struct tile_info info,
```

Comment:
Pass by reference to avoid a copy, and no need to repeat `struct` when not ambiguous:
```suggestion
const char *tile_get_info_text(const tile_info &info,
```

Reply:
Done